docs/development_guide.md

```diff
@@ -131,9 +133,10 @@ $ # Create docker secret in Kubernetes Namespace using following script:
 $ ./scripts/create-container-registry-creds.sh "$K8S_NAMESPACE"
 
 ```
+(NOTE: these requirements are true for automatic demo deployment using the available demo scripts, #1416 tracks an improvement to not strictly require this and use upstream images from official dockerhub registry if a user does not want/need changes on the code)
```

Comment:
```suggestion
(NOTE: these requirements are true for automatic demo deployment using the available demo scripts, #1416 tracks an improvement to not strictly require these and use upstream images from official dockerhub registry if a user does not want/need changes on the code)
```

Reply:
hm... guess with the new remote fork workflow we lost he ability to commit on the local pr branch. Is it normal to not be able to commit suggestions?

![image](https://user-images.githubusercontent.com/5503924/89492885-8fa85b00-d767-11ea-9e2d-94befb8c7def.png)

Reply:
Right, we should not using branches in the repo at this point. We should PR from our forks.

Reply:
+1. @eduser25 please create a PR from a fork.